packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -81,7 +89,7 @@ export class H5GroveApi extends DataProviderApi {
       searchParams.set('selection', selection);
     }
 
-    return `${baseURL as string}/data/?${searchParams.toString()}`;
+    return new URL(`${baseURL as string}/data/?${searchParams.toString()}`);
```

Comment:
You could set the base separately ``new URL(`data/?${searchParams.toString()}`, baseURL as string)`` but I don't mind the current form

Reply:
Seems that `baseURL` gets truncated down to the origin (protocol + domain) when it is passed as second argument to `new URL()`: the path `/h5grove` is removed. I've reverted back to what I had initially. Good try though.

Reply:
[Indeed](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL):
```js
new URL('/en-US/docs', "https://developer.mozilla.org/fr-FR/toto"); 
// => 'https://developer.mozilla.org/en-US/docs'
```

Well good to know.